src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,817 @@
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('35');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowRight' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('40');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'Home' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('0');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'End' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('75');
+});
+
+it('should limit keystrokes processing by min max when allow-crossing is set', () => {
+  const handleOnChange = jest.fn();
+  const wrapper = render(
+    <Slider
+      values={[40, 80]}
+      step={5}
+      setFocus
+      thumbMode='allow-crossing'
+      onChange={handleOnChange}
+    />,
+  );
+  const { container } = wrapper;
+  assertBaseElement(container);
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+
+  expect(document.activeElement).toEqual(thumb);
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('40');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowLeft' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('35');
+
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'ArrowRight' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('40');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'Home' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('0');
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'End' });
+  });
+  thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('100');
+  expect(handleOnChange).toHaveBeenCalledTimes(4);
+
+  // triggering an update with same value should not trigger callback
+  act(() => {
+    fireEvent.keyDown(thumb, { key: 'End' });
+  });
+  expect(handleOnChange).toHaveBeenCalledTimes(4);
+});
+
+it('should NOT process keystrokes when slider is disabled', () => {
+  const wrapper = render(<Slider values={[50]} step={5} disabled />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  let thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+
+  // force thumb to have focus
+  thumb.focus();
```

Comment:
Force thumb to have focus? It has no tabindex, so shouldn't get focus.

What does `document.activeElement` print here?